contracts/RealmDiamond/facets/RealmGettersAndSettersFacet.sol

```diff
@@ -1,138 +1,161 @@
-// SPDX-License-Identifier: UNLICENSED

```

Comment:
Why is this whole section being removed and replaced? I cannot review code like this @orionstardust

Reply:
@cinnabarhorse I think, This is GitHub issue, I think.